### PR TITLE
Burotel: Change splash image

### PR DESCRIPTION
**Pull request by meluru**

### CODE REVIEW

**Comment by ThiefMaster** [comment]
I think a 5MB PNG is a bit excessive - let's use a JPEG for this.

**Comment by meluru** [comment]
Updated

File: burotel/indico_burotel/client/styles/palette.scss [hunk 1]
```diff
@@ -14,5 +14,5 @@ $dark-highlight-color: #b1604b;
 $primary-color: #2185d0;
 
 %rb-splash-background {
-    background-image: url('burotel:images/splash.png');
+    background-image: url('burotel:images/splash.jpeg');
```
Comment (ThiefMaster) [inline burotel/indico_burotel/client/styles/palette.scss]: .jpg - jpeg is a very uncommon extension even if that's the name of the format ;)

**Comment by meluru** [comment]
It was very huge before and I checked that the globe in Indico is even smaller  1920 x 815

**Comment by meluru** [comment]
Updated

**Pull request merged**